src/main/java/org/openrewrite/kotlin/internal/KotlinTreeParserVisitor.java

```diff
@@ -2892,16 +2892,28 @@ public J visitProperty(KtProperty property, ExecutionContext data) {
         }
 
         if (getter != null || setter != null || receiver != null || typeConstraints != null) {
+            List<JRightPadded<J.MethodDeclaration>> rps = new ArrayList<>(2);
+
+            if (setter != null) {
+                rps.add(padRight(setter, Space.EMPTY));
+            }
+
+            if (getter != null) {
+                rps.add(padRight(getter, Space.EMPTY));
+            }
+
+            if (!isSetterFirst) {
+                Collections.reverse(rps);
```

Comment:
It seems like it would be simpler to just use `KtProperty#getAccessors()` which should retrieve them in the order they occur in the source code.

Reply:
yeah, good point, `KtProperty#getAccessors()` can make it simpler. revised to use `KtProperty#getAccessors()`